README.md

```diff
@@ -770,6 +770,25 @@ const client = createClient({
 
 </details>
 
+<details id="kotlin">
+<summary><a href="#kotlin">🔗</a> Client usage with <a href="https://github.com/apollographql/apollo-kotlin">Apollo Kotlin</a></summary>
+
+Connect to [`graphql-ws`](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) compatible server in Kotlin using [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin)
```

Comment:
FWIW, most issues on the Kotlin repo mention `graphql-ws` as in the library name and very few mention the subprotocol. Did you consider renaming the repo to `graphql-transport-ws` to make both consistent?

Reply:
It _was_ named `graphql-transport-ws` before. But I've acquired the name `graphql-ws` and [since v1.9.1 that's the name I stuck with](https://github.com/enisdenjo/graphql-ws/releases/tag/v1.9.1). It's shorter and easier.

Nevertheless I own the `graphql-transport-ws` lib on npm + this repo was named `graphql-transport-ws` before; meaning, `graphql-ws` can be found through that name.
- https://www.npmjs.com/package/graphql-transport-ws
- https://github.com/enisdenjo/graphql-transport-ws

Reply:
Nevertheless, I understand the confusion, which is why I was pushing the `graphql-transport-ws` name.

P.S. I guess there'll be a time when s-t-p is part of history.